domains/courier.json

```
@@ -0,0 +1,12 @@
+{
+        "owner": {
+           "username": "couriervn",
+           "email": "2familan2011@Gmail.com",
+           "discord": "689407629709410314"
+        },
+    
+        "record": {
+            "CNAME": "https://couriervn.github.io/"
```

```suggestion
            "CNAME": "couriervn.github.io"
```
Also please add more stuff to your website